app/views/layouts/steps_to_become_a_teacher.html.erb

```diff
@@ -42,6 +48,7 @@
         <%= render "components/videoplayer" %>
         <%= render "sections/cookie-acceptance" %>
         <%= render "sections/feedback-bar" %>
+        <%= render "components/analytics" %>
```

Comment:
Would rendering this at the top allow us to avoid passing the global frontmatter around?

Reply:
I almost took that route, but there are other things that depend on the front matter further down, such as:

https://github.com/DFE-Digital/get-into-teaching-app/blob/73b6992f8726590012646ade57a0abb8c65bfde8/app/views/layouts/steps_to_become_a_teacher.html.erb#L43

Reply:
Ok. I think we're almost in a position to remove `hide_page_helpful_question` and render it based on the template, perhaps we can adjust this at a later date 👍🏽